### DISCUSSION TITLE
Where di the Jupyter Notebooks go?

### DISCUSSION THREAD

**Discussion by byronamerson** [modflowpy/flopy#1813] [Q&A] [answered]
Hello most excellent flopy team. I was navigating to a notebook yesterday and notice that link seems to be broken.

https://github.com/modflowpy/flopy/blob/develop/docs/notebook_examples.md

Using this link above  leads me to a "404- Page Not Found error. Screenshot below

![image](https://github.com/modflowpy/flopy/assets/63072063/1cf1b160-f313-4156-b9a5-735ecd8697a1)

Best, Byron

**Comment by jlarsen-usgs** [comment, marked as answer]
@byronamerson 

Notebooks have been moved into the FloPy documentation. They can be viewed and downloaded from here [https://flopy.readthedocs.io/en/latest/examples.html](https://flopy.readthedocs.io/en/latest/examples.html). The individual python scripts that are used to generate the notebooks can also be found at [https://github.com/modflowpy/flopy/tree/develop/.docs/Notebooks](https://github.com/modflowpy/flopy/tree/develop/.docs/Notebooks)
